### DISCUSSION TITLE
Make each workspace button a different color

### DISCUSSION THREAD

**Discussion by RGBCube** [Alexays/Waybar#2696] [Q&A] [answered]
as the title says, what elements do i have to style?

**Comment by MajorMuff** [comment, marked as answer]
I personally style each button separately using css' ":nth-child()" selector which seems to work well. Find it in my style.css [here](https://github.com/MajorMuff/arch-hypr/blob/main/dotfiles/waybar/style.css)
I import the colors from a separate css file which in turn I update using a script when I feel like changing things up.

There's probably some better way of doing this but this works for me.
